### PR TITLE
Blockcerts Edit

### PR DESCRIPTION
**Pull request by shannona**

### CODE REVIEW

File: draft-documents/BlockcertsV3.md [hunk 1]
```diff
@@ -671,19 +666,17 @@ An alternative method of making this `BlockcertsRevocationService` would be to m
 }]
 ```
 
-There are pros and cons to both. 
-
-Option 1 allows a verifier to only obtain and see the revocation status of a single certificate. A verifier would not be able to see the revocation status of certificates in which they do not have possession of. A standard practice of making UUID-based certificate IDs should prohibit verifiers from guessing another certificate ID. 
+There are pros and cons to both options. 
 
-However, option 1 allows the issuer to see, log, monitor (etc.) against a specific certificate. They would be able to see what IP address, origin, etc. was trying to verify a specific individual's certificate and then infer certain things. 
+Option 1 allows a verifier to only obtain and see the revocation status of a single certificate. A verifier would not be able to see the revocation status of certificates that do not possess. A standard practice for making UUID-based certificate IDs should prohibit verifiers from guessing another certificate ID.
```
Comment (ghost) [inline draft-documents/BlockcertsV3.md]: "that _they_ do not possess"?

File: draft-documents/BlockcertsV3.md [hunk 2]
```diff
@@ -671,19 +666,17 @@ An alternative method of making this `BlockcertsRevocationService` would be to m
...
 
-There has not been a very good consensus yet on what method of revocation/status lists should be used for Verifiable Credentials, and thus no standards yet. Ideally, there is a generic `RevocationServiceEndpoint` not specific to Blockcerts revocation lists, but to not conflict with other methods that might be created, we are suggesting we label this as a Blockcerts-specific revocation endpoint. 
-
-Instead of the Blockcerts standard picking one of these two methods, we may support both and allow issuers to decide for themselves which makes better sense for their organization. 
+There has not been a very good consensus yet on what method of revocation/status lists should be used for Verifiable Credentials, and thus there are no standards yet. Ideally, there is a generic `RevocationServiceEndpoint` not specific to Blockcerts revocation lists, but that does not conflict with other methods that might be created. We suggest that we label this as a Blockcerts-specific revocation endpoint.
```
Comment (ghost) [inline draft-documents/BlockcertsV3.md]: > Ideally, there is a generic `RevocationServiceEndpoint` not specific to Blockcerts revocation lists, but that does not conflict with other methods that might be created. We suggest that we label this as a Blockcerts-specific revocation endpoint. 

Maybe: 

"Ideally, there is a generic `RevocationServiceEndpoint` for all Verifiable Credential revocations. In the meantime, we suggest that we label this as a Blockcerts-specific revocation endpoint."

**Review by ghost — CHANGES_REQUESTED** [review]
Thank you @shannona for those edits, I commented on two small parts, please take a look whenever you have a chance.

**Comment by shannona** [comment]
Thanks for the fixes, @AnthonyRonning. I've incorporated them, and am incorporating the PR.

**Comment by ghost** [comment]
Perfect, thank you so much!

**Pull request merged**